src/commands/components/init/component.ts

```diff
@@ -116,6 +116,7 @@ export default class GenerateComponentCommand extends Command {
       scripts: {
         build: "webpack",
         publish: "npm run build && prism components:publish",
+        "generate:manifest": "npm run build && npx @prismatic-io/spectral component-manifest",
```

Comment:
```suggestion
        "generate:manifest": "npm run build && npx @prismatic-io/spectral component-manifest --include-signature",
        "generate:manifest:dev": "npm run build && npx @prismatic-io/spectral component-manifest",
```
@bkegley do we want to provide them with a command that always generates a signature (if component published anyways) and another for purely development so the end user doesn't have to worry about the flags.